lease/lessor_test.go

```diff
@@ -47,16 +63,21 @@ func TestLessorGrant(t *testing.T) {
 // the DeleteableKV.
 // The revoked lease cannot be got from Lessor again.
 func TestLessorRevoke(t *testing.T) {
+	dir, be := NewTestBackend(t)
+	defer be.Close()
+	defer os.RemoveAll(dir)
```

Comment:
it does not really matter. OS will only release the resource when all the file descriptor are released. But yes, I can change to make this read better.